holdem/env.py

```diff
@@ -219,17 +219,16 @@ def step(self, actions):
         self._current_player = self._next(players, self._current_player)
         players.remove(folded_player)
         self._folded_players.append(folded_player)
-        # break if a single player left
-        if len(players) == 1:
-          self._resolve(players)
+        # break if a single player left <<-- already will be resolved in next check, if we folded then there must be a bet and the other player has acted
+        #if len(players) == 1:
```

Comment:
Code should be not commented unless there is a good reason for it - if those lines are truly not necessary, then just remove them.

Reply:
Removed it